src/containers/Landing/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styled from 'styled-components'
+import Banner from '../../components/Banner'
+import logo_lockup from '../../assets/logo_lockup.svg'
+import holo from '../../assets/holo.svg'
+import holo_end from '../../assets/holo2.svg'
+import Footer from './Footer'
+import { H1, P } from '../../components/Typography'
+import { CenterHorizontally } from '../../components/Common'
+
+const LandingContainer = styled.div`
+  overflow-x: hidden;
+  padding: 0 auto;
+`
+
+const HoloBackground = styled.img`
+  position: absolute;
+  float: left;
+  overflow-x: hidden;
+  display: block;
+  opacity: 0.8;
+  width: 100%;
+  top: 18em;
+  ${CenterHorizontally}
+  z-index: -1;
+`
+
+const StyledLogoLockup = styled.img`
+  position: absolute;
+  top: 7em;
+  margin: 0 50.5%;
```

Comment:
Note about this 50.5... the logo was not perfectly symmetrical by default so a 0.5 offset is needed.

#### before
<img width="431" alt="Screen Shot 2020-11-27 at 12 30 15 AM" src="https://user-images.githubusercontent.com/38872354/100432768-e73c4080-304e-11eb-909f-2a888bc52263.png">

#### after
<img width="709" alt="Screen Shot 2020-11-27 at 12 30 29 AM" src="https://user-images.githubusercontent.com/38872354/100432553-9c222d80-304e-11eb-8c4c-eaa6133df0d2.png">